packages/account/src/Sections/Verification/ProofOfOwnership/__test__/card.spec.js

```diff
@@ -0,0 +1,23 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import Card from '../card.jsx';
+import React from 'react';
+import { grouped_payment_method_data } from './test-data';
+
+describe('card.jsx', () => {
+    it('Should render a card', () => {
+        render(<Card card={grouped_payment_method_data.visa} />);
+        expect(
+            screen.getByTestId(grouped_payment_method_data.visa.payment_method, { exact: true })
```

Comment:
as I understand, second parameter to queries helps adjust us the query with string: whether to use part of a string or full, to skip some part or not.  It is better to adjust it with RegExp, and also { exact: true } is a default value. So there is no need to use this second param in queries.
Could you please also check the same for other cases in tests

Reply:
@yauheni-kryzhyk-deriv, just to be clear, do you want me to remove `{ exact: true }` ? 😄